src/main/java/nextstep/subway/path/dto/Path.java

```diff
@@ -0,0 +1,69 @@
+package nextstep.subway.path.dto;
+
+import nextstep.subway.line.entity.Line;
+import nextstep.subway.path.exception.PathException;
+import nextstep.subway.section.entity.Section;
+import nextstep.subway.station.dto.StationResponse;
+import nextstep.subway.station.entity.Station;
+import org.jgrapht.GraphPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static nextstep.subway.common.constant.ErrorCode.PATH_NOT_FOUND;
+
+public class Path {
+    private final List<Long> vertexList;
```

Comment:
```suggestion
    private final List<Stations> stations;
```

Path의 vertexList보다는, stations를 필드로 가져가 보는 건 어떨까요?